checkov/cloudformation/runner.py

```diff
@@ -63,5 +73,44 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                                                 evaluations=variable_evaluations,check_class=check.__class__.__module__,
                                                 file_abs_path=file_abs_path, entity_tags=tags)
                                 report.add_record(record=record)
+
+        # run graph checks only if environment variable CHECKOV_CLOUDFORMATION_GRAPH='true'
+        should_create_graph = os.environ.get("CHECKOV_CLOUDFORMATION_GRAPH")
+        if should_create_graph and should_create_graph.lower() == "true":
+            logging.info("creating cloudformation graph")
+            local_graph = self.graph_manager.build_graph_from_definitions(self.definitions)
+            self.graph_manager.save_graph(local_graph)
+            graph_report = self.get_graph_checks_report(root_folder, runner_filter)
+            merge_reports(report, graph_report)
+
         return report
 
+    def get_graph_checks_report(self, root_folder, runner_filter: RunnerFilter):
+        report = Report(self.check_type)
+        checks_results = self.run_graph_checks_results(runner_filter)
+
+        for check, check_results in checks_results.items():
+            for check_result in check_results:
+                entity = check_result['entity']
+                entity_file_abs_path = create_file_abs_path(root_folder, entity.get(CustomAttributes.FILE_PATH))
+                entity_name = entity.get(CustomAttributes.BLOCK_NAME).split(".")[1]
+                entity_context = self.context.get(entity_file_abs_path).get(CloudformationTemplateSections.RESOURCES).get(entity_name)
```

Comment:
if you try to retrieve the corresponding values, without any default, then it can result in ugly `NoneType` errors. I think it would be better just to directly access the value, if they will be there for sure via `entity[key]`.